house/house.go

```diff
@@ -143,6 +151,36 @@ func NewServer() HomeManager {
 	return server
 }
 
+func (s *Server) Devices(w http.ResponseWriter, req *http.Request) {
```

Comment:
warning: exported method Server.Devices should have comment or be unexported (golint)